ui/src/main/webapp/src/app/components/project-list/project-list.component.scss

```diff
@@ -72,6 +47,14 @@ a {
     .project-info:hover { background-color: rgb(222, 243, 255); }
 }
 
+/* Override Patternfly's middle alignment. */
+.list-view-pf.projects-list .list-group-item.project-info,
+.list-view-pf.projects-list .list-view-pf-description { align-items: flex-start; }
+
+/* Action buttons padding */
+.list-view-pf.projects-list .list-view-pf-actions span { margin-left: 1em; }
+.list-view-pf.projects-list .list-view-pf-actions span a i { font-size: 16pt; }
```

Comment:
I think it's better this way. We can hold the PR until some UX expert decides, or we can merge it now and change if there are objections.